backend/local/backend.go

```diff
@@ -407,3 +408,22 @@ func (b *Local) stateWorkspaceDir() string {
 
 	return DefaultWorkspaceDir
 }
+
+func (b *Local) pluginInitRequired(providerErr *terraform.ResourceProviderError) {
+	b.CLI.Output(b.Colorize().Color(fmt.Sprintf(
+		strings.TrimSpace(errPluginInit)+"\n",
+		providerErr)))
+}
+
+// this relies on multierror to format the plugin errors below the copy
+const errPluginInit = `
+[reset][bold][yellow]Plugin reinitialization required. Please run "terraform init".[reset]
+[yellow]Reason: Could not satisfy plugin requirements.
+
+Plugins are external binaries that Terraform uses to access and manipulate
+resources. If this message is showing up, it means that the configuration you
```

Comment:
Is there an alternative wording of this that doesn't include "If this message is showing up"? It feels a little weird to include that, since by definition the message _is_ showing up if you're reading this.

Perhaps just "The configuration you have requires plugins which can't..."